# Timestamp
Header header

int32[] AnchorID 

float64[] x
float64[] y
float64[] z

float64[] tdoa_of_anchors
